pkg/controllers/machine_controller.go

```diff
@@ -554,6 +561,21 @@ func (r *MachineReconciler) domainFor(
 	return domainDesc, volumeStates, nicStates, nil
 }
 
+func (r *MachineReconciler) setDomainMetadata(machine *api.Machine, domain *libvirtxml.Domain) error {
+	domainMetadata := &providermeta.LibvirtProviderMetadata{
+		Namespace: machine.Metadata.Labels[ironcorev1alpha1.MachineNamespaceLabel],
```

Comment:
what will be result, if namespace isn't defined in labels?

Reply:
It will be empty.

Reply:
yes i know, but are you okay with that? because map return 2 values (value for key and boolean if key exists)

Reply:
we had call. if label is missing, it isn't problem of this part of code